binding.gyp

```diff
@@ -39,8 +39,10 @@
         }],
         ['OS=="mac" or OS=="solaris"', {
           'xcode_settings': {
-            'GCC_ENABLE_CPP_EXCEPTIONS': 'YES'
+            'GCC_ENABLE_CPP_EXCEPTIONS': 'YES',
+            'MACOSX_DEPLOYMENT_TARGET': '10.6',
```

Comment:
Maybe-ish. I think it should be safe building for at least Snow Leopard for a little (long?) while.